fluentd/run.sh

```diff
@@ -109,15 +109,84 @@ else
 fi
 export K8S_FILTER_REMOVE_KEYS
 
+if [ -z $ES_HOST ]; then
+    echo "ERROR: Environment variable ES_HOST for Elasticsearch host name is not set."
+    exit 1
+fi
+if [ -z $ES_PORT ]; then
+    echo "ERROR: Environment variable ES_PORT for Elasticsearch port number is not set."
+    exit 1
+fi
+
+OPS_COPY_HOST="${OPS_COPY_HOST:-$ES_COPY_HOST}"
+OPS_COPY_PORT="${OPS_COPY_PORT:-$ES_COPY_PORT}"
+OPS_COPY_SCHEME="${OPS_COPY_SCHEME:-$ES_COPY_SCHEME}"
+OPS_COPY_CLIENT_CERT="${OPS_COPY_CLIENT_CERT:-$ES_COPY_CLIENT_CERT}"
+OPS_COPY_CLIENT_KEY="${OPS_COPY_CLIENT_KEY:-$ES_COPY_CLIENT_KEY}"
+OPS_COPY_CA="${OPS_COPY_CA:-$ES_COPY_CA}"
+OPS_COPY_USERNAME="${OPS_COPY_USERNAME:-$ES_COPY_USERNAME}"
+OPS_COPY_PASSWORD="${OPS_COPY_PASSWORD:-$ES_COPY_PASSWORD}"
+ES_COPY_SSL_VERIFY="${ES_COPY_SSL_VERIFY:-true}"
+export OPS_COPY_HOST OPS_COPY_PORT OPS_COPY_SCHEME OPS_COPY_CLIENT_CERT \
+       OPS_COPY_CLIENT_KEY OPS_COPY_CA OPS_COPY_USERNAME OPS_COPY_PASSWORD \
+       ES_COPY_SSL_VERIFY
+
 # How many outputs?
 if [ -n "${MUX_CLIENT_MODE:-}" ] ; then
     # A fluentd collector configured as a mux client has just one output: sending to a mux.
     NUM_OUTPUTS=1
+    if [ "$ES_COPY" = "true" ] ; then
+        echo "WARNING: When MUX_CLIENT_MODE is set, logs are forwarded to MUX; COPY won't work with it."
+    fi
 else
-    # fluentd usually has 2 outputs.
-    NUM_OUTPUTS=2
+    # check ES_HOST vs. OPS_HOST; ES_PORT vs. OPS_PORT
+    if [ "$ES_HOST" = ${OPS_HOST:-""} -a $ES_PORT -eq ${OPT_PORT:-0} ]; then
+        # There is one output Elasticsearch
```

Comment:
But there still may be two outputs if using secure_forward to copy logs to outside of the cluster